annotations/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```

Comment:
https://gradle.org/releases/

Are there specific reasons behind to use version 7.5.1? 

The latest version is 8.1.1 (according to https://gradle.org/releases/), which have in fact slightly different public contract than v7.

If we want to use v7 anyway, then I will suggest to use the latest (v7.6.1)